rts/Map/Ground.cpp

```diff
@@ -588,12 +591,27 @@ float CGround::TrajectoryGroundCol(const float3& trajStartPos, const float3& tra
 		const float3 pos = (trajStartPos + dir * dist) + (alt * dist * dist);
 
 		#if 1
-		if (GetApproximateHeight(pos) > pos.y)
+		if (GetApproximateHeight(pos) > pos.y) {
+			if (globalRendering->drawDebugTraceRay) {
+
+				#define go geometricObjects
+
+				// red line pointing to hitPos
+				go->SetColor(go->AddLine(pos, trajStartPos, 3, 0, GAME_SPEED), 1.0f, 0.0f, 0.0f, 1.0f);
```

Comment:
the `#define` looks very meek?
```suggestion
				geometricObjects->SetColor(geometricObjects->AddLine(pos, trajStartPos, 3, 0, GAME_SPEED), 1.0f, 0.0f, 0.0f, 1.0f);
```